### ISSUE TITLE
Switch buttons background images not loaded via reverse proxy

### ISSUE THREAD

**Issue by frepkovsky** [domoticz/domoticz#5389]
When running Domoticz behind reverse proxy, selector switch level buttons are not loaded properly when using default theme:

Domoticz dashboard when accessed directly - all OK:

![image](https://user-images.githubusercontent.com/51033177/199024405-b91243c6-c60c-4299-8860-d9fe98ef8165.png)

Domoticz dashboard when accessed via reverse proxy - some elements are missing:

![image](https://user-images.githubusercontent.com/51033177/199024493-4991a066-7da4-45fe-b02c-501049bd5f7d.png)

from reverse proxy logs:

```
# tail -f error.log
2022/10/31 14:22:24 [error] 10140#0: *253 open() "/opt/html/css/images/img03b.jpg" failed (2: No such file or directory), client: 192.168.0.100, server: , request: "GET /css/images/img03b.jpg HTTP/1.1", host: "192.168.0.10:8443", referrer: "https://192.168.0.10:8443/domoticz/styles/default/custom.css"
```

Cause of the issue is absolute path used in url for button background images in default theme custom.css - `www/styles/default/custom.css`. When changed to relative path all is loaded properly when accessing Domoticz dashboard directly or via proxy.

**Comment by kiddigital** [comment]
@frepkovsky , if using the default theme, you should be able to run the UI served directly by the proxy itself (the `www` folder). Does __not__ work when using non-default theme as the Domoticz webserver does specific processing for these.

You _only_ need to proxy a few requests, see [the lighttpd config](https://github.com/domoticz/domoticz/blob/development/test/lighttpd.conf) for details.

Often, a proxy is used as a _first_ line of defense to protect underlying services, do SSL offloading for example, etc. To improve security, only proxy-ing a very specific small set of request to the underlying service (Domoticz in this case) makes the attack surface much, much smaller and also protects the Domoticz webserver against a lot of attack attempts.

But of course proxy-ing everything works :)

**Comment by frepkovsky** [comment]
@kiddigital, thanks for the hint about possibility to serve static UI content by proxy itself and proxying the API calls only. Yes, I started simply with proxying everything ;-). I realized that thanks to more and more protocol integrations I use in Domoticz (which is great and maybe that's I like the most on Domoticz), I have more and more http services listening on network (Domoticz UI, ZwaveJS UI, Zigbee2mqtt UI, Mosquitto broker, ...) plus some other apps out of Domoticz so I would like to secure them all and restrict access via application gateway (reverse proxy) where I can manage users, x509 sso etc. I do not plan to expose the proxy to the internet, just trying to improve security my home network ;)
